bullet_train-themes-light/app/views/showcase/engine/_head.html.erb

```diff
@@ -36,11 +36,13 @@
   document.addEventListener("DOMContentLoaded", () => {
     const node = document.getElementById("bullet_train_theme_selects").content.cloneNode(true)
     document.querySelector("main").appendChild(node)
+    const selectorNode = document.getElementById("bt-theme-selector")
+    document.querySelector("main").style.paddingBottom = selectorNode.offsetHeight + "px"
   })
 </script>
 
 <template id="bullet_train_theme_selects">
-  <section class="border-t w-full p-4 flex justify-end left-0 bottom-0">
+  <section class="border-t w-full p-4 flex justify-end left-0 bottom-0 fixed bg-white dark:sc-bg-neutral-900" id="bt-theme-selector">
```

Comment:
Interesting, it didn't look to me like we needed `bg-white dark:sc-bg-neutral-900` because we got `bg-inherit` automatically from being in `main`. Also `sc-bg-neutral-900` isn't safe since that comes from Showcase's CSS build and would go away if Showcase stopped using it.

Reply:
Without those that div is transparent and content shows through it.

![CleanShot 2023-09-01 at 12 58 01](https://github.com/bullet-train-co/bullet_train-core/assets/58702/7ffd2bbb-0421-45bd-b721-79eb851caee7)

Reply:
Dang, wut. It seems like we don't really ship dark-mode colors, I tried `dark:bg-black` but that's kinda too much. I don't know if there's a way to transform colors in Tailwind, it would be nice if we didn't have to muck with the build or pollute application.css.

Reply:
I couldn't even get `dark:bg-black` to do anything. It seemed to fallback to `bg-white`, which is no good with white text.